index.html

```diff
@@ -359,13 +372,16 @@ <h4>Marta Pieńkowska</h4>
                     <span>Poczatkująca Front-End developerka. Wielbicielka mango i czekolady. W wolnych chwilach lubi spędzać czas z przyjaciółmi, siedziec na słońcu i cieszyć się chwilą.</span>
                     <div class="social-icons">
                         <div class="git-hub social">
-                            <a href="https://github.com/PienkowskaMarta" target="_blank"><img class="team-img" src="img/github-logo.png" alt=""></a>
+                            <a href="https://github.com/PienkowskaMarta"><img class="team-img"
+                                                                                              src="img/github-logo.png"
+                                                                                              alt=""></a>
                         </div>
                         <div class="linkedin social">
-                            <a href="https://pl.linkedin.com/in/marta-pie%C5%84kowska-7b173454" target="_blank"><img class="team-img" src="img/linkedin.png" alt=""></a>
+                            <a href="https://pl.linkedin.com/in/marta-pie%C5%84kowska-7b173454"><img
+                                    class="team-img" src="img/linkedin.png" alt=""></a>
                         </div>
                         <div class="portfolio social">
-                            <a href="" target="_blank"><img class="team-img" src="img/resume.png" alt=""></a>
+                            <a href=""><img class="team-img" src="img/resume.png" alt=""></a>
```

Comment:
napewno trzeba wyrzucic "target_blank"? jak linki beda sie otwierac?

Reply:
czemu wyrzuciłaś target blank??, to musi być, inaczej link otworzy się zamiast naszej strony as powinien w nowej zakładce

Reply:
to Pawla PR